chrome/content/zotero/xpcom/data/item.js

```diff
@@ -1274,7 +1274,7 @@ Zotero.Item.prototype._saveData = Zotero.Promise.coroutine(function* (env) {
 		yield Zotero.DB.queryAsync(sql, env.sqlValues);
 		
 		if (!env.options.skipNotifier) {
-			Zotero.Notifier.queue('modify', 'item', itemID, env.notifierData);
+			Zotero.Notifier.queue('modify', this._objectType, itemID, env.notifierData);
```

Comment:
OK. Various other things also expect 'item', but maybe not anything we actually want to run for feed items. Can just keep an eye out for glitches.